src/main/java/org/springframework/integration/aws/outbound/SnsMessageHandler.java

```diff
@@ -38,6 +39,7 @@
 import com.amazonaws.services.sns.model.MessageAttributeValue;
 import com.amazonaws.services.sns.model.PublishRequest;
 import com.amazonaws.services.sns.model.PublishResult;
+
```

Comment:
This is something what is not expected as a change.
`com`, `io` etc. have to be grouped together.

Reply:
The existing file has `org.springframework` and `com.amazonaws` separated by a blank line, and it appears the Eclipse default grouping also added a separation between `com.amazonaws` and `io.awspring`. Should I retain the first (existing) blank line and remove the second (new) one?